tests/e2e/utils/network_utils.go

```diff
@@ -394,34 +422,37 @@ func WaitGetPIP(azureTestClient *AzureTestClient, ipName string) (pip aznetwork.
 	return
 }
 
-// SelectAvailablePrivateIP selects a private IP address in Azure subnet.
-func SelectAvailablePrivateIP(tc *AzureTestClient) (string, error) {
+// SelectAvailablePrivateIPs selects private IP addresses in Azure subnet.
+func SelectAvailablePrivateIPs(tc *AzureTestClient) ([]string, error) {
```

Comment:
Can we unit test this one?

Reply:
Yes. I turned "selectSubnets" and "findIPInSubnet" parts into 2 functions. Then I added a UT for the first one. As for the second one, the logic is the same as before and I think it is already clear. PTAL.